nomos-cli/src/cmds/validator.rs

```diff
@@ -29,6 +30,14 @@ pub struct Retrieve {
     pub addr: Url,
 }
 
+#[derive(Args, Debug)]
+pub struct Reconstruct {
+    /// Blobs to use for reconstruction.
+    #[clap(short, long)]
+    pub app_blobs: String,
```

Comment:
Can we use another type that is a bit more self descriptive for this? So basically now the cli expects a json string that was received from the validator api. Maybe it would be better to read is from the file?

Reply:
Yeah, definetely not a `String`, if it is a `json` at least a `json::Value` (IIRC).

Reply:
Replaced at https://github.com/logos-co/nomos-node/pull/994/commits/4e2f7617c944f9fac83173ea8ac383374b70d459

Reply:
How would you like to have the input here ? I changed back to JSON string. It reads from file. I could add more description in the help message ? Or is there any example how it could be done in more polished way ?